### PR TITLE
feature/custom-tab-itens-component

### PR DESCRIPTION
**Pull request by lucasvps**
_Changes:_

_1. What i did:_

- Completed the [task](https://www.notion.so/5e06e62f649b4d5c8a2f40a85f74c771?v=14e3d2c090ca484db4b6e2f077a7347a&p=09afed8a214842cf8e63a62acf38dd2e&pm=s)
- Create a Custom Tab Item Component

_2. What to test:_

- [ ] Layout on different devices ([mockup](link_to_mockup))
- [ ] Happy path
- [ ] Error handling

---

_How to get there_

_1. Flavor and login informations_

> _flavor_: staging

_2. Steps_

- Run Example App
- Click on Example Screen Button

---

_Observations:_

_1. Known issues_

- No design to based yet

_2. Code review pratices_

- Unecessary prints/log (ctrl+f -> [ print( ] )
- Unecessary comments (ctrl+f -> [ // ] )
- Code organization
- Naming of functions and variables
- Repeated code

### CODE REVIEW

**Review by gabrielerego123 — APPROVED** [review]

File: example/lib/screens/example_screen.dart [hunk 1]
```diff
@@ -1,6 +1,9 @@
+import 'dart:developer';
```
Comment (ItaloDias-stack) [inline example/lib/screens/example_screen.dart:1]: Remover essa tela

File: lib/widgets/custom_tab_items.dart [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        }
+      });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: widget.backgroundDecoration,
+      width: MediaQuery.of(context).size.width,
+      // height: 40,
```
Comment (ItaloDias-stack) [inline lib/widgets/custom_tab_items.dart]: Remover comentário

**Review by ItaloDias-stack — CHANGES_REQUESTED** [review]

File: example/lib/screens/example_screen.dart (hunk 1 above)
Comment (lucasvps) [reply example/lib/screens/example_screen.dart:1]: Por que exatamente remover essa tela? @ItaloDias-stack

File: example/lib/screens/example_screen.dart [hunk 3]
```diff
@@ -32,6 +35,33 @@ class _ExampleScreenState extends State<ExampleScreen> {
           controller: controller,
           child: Column(
             children: [
+              CustomTabItens(
```
Comment (arkaisho) [inline example/lib/screens/example_screen.dart]:
```suggestion
              CustomTabItems(
```

**Review by arkaisho — CHANGES_REQUESTED** [review]

**Pull request merged**